recipes/cpr/all/conanfile.py

```diff
@@ -33,6 +33,10 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+        if self.options.with_openssl:
+            if self.options["libcurl"].with_openssl == False:
```

Comment:
Replace with with `if self.options["libcurl"].with_openssl is False:` (**is**) because my ide wants to simplify this line.

Reply:
Also, add a comment here stating that `self.options["libcurl"].with_openssl` can be None when it is defaulted.
Would a `self.output.info` help here?

Reply:
I'm sorry for your IDE, but I cannot change that line. The object behind the options doesn't behave the same with `is` or `==`. I did an analysis a long time ago at https://github.com/conan-io/conan/issues/3620.

About the other question, I think that a comment should be enough, it's a message for the recipe developer, not for the consumer, I think.

Reply:
Thanks for the link.
Too bad the behavior is different from what you would expect from a bool python object.

Reply:
I've added it to the milestone for Conan 2.0. That's something important to consider.